v5.8.2 (#32949)
mui/material-ui: CHANGELOG.md

# [Versions](https://mui.com/versions/)

## 5.8.2

<!-- generated comparing v5.8.1..master -->

_May 30, 2022_

> A big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
> 
> - 🐛 bug fixes and 📚 documentation improvements.

Reviewer: How about something along these lines:

```suggestion
A big thanks to the 8 contributors who made this release possible. It is mostly about 🐛 bug fixes and 📚 documentation improvements.
```

### `@mui/material@5.8.2`

> - &#8203;<!-- 04 -->[system] Add `getColorSchemeSelector` util (#32868) @siriwatknp

Reviewer: Shouldn't this go to @mui/system?

Author: Updated.

Reviewer: This means we don't have anything to release in Material UI this time 😮

### `@mui/lab@5.0.0-alpha.84`

> - &#8203;<!-- 07 -->[Masonry] Place items to the left when there is less object than specified in `column` prop (#32873) @hbjORbj

Reviewer 2: ```suggestion
- &#8203;<!-- 07 -->[Masonry] Place items to the left when there are less objects than specified in `column` prop (#32873) @hbjORbj
```

### `@mui/base@5.0.0-alpha.83`

> - &#8203;<!-- 24 -->[BadgeUnstyled] Define ownerState and slot props' types (#32750) @michaldudak

Reviewer 2: @michaldudak was on fire last week :D

- &#8203;<!-- 06 -->[SliderUnstyled] Define ownerState and slot props' types (#32739) @michaldudak
- &#8203;<!-- 05 -->[SwitchUnstyled] Define ownerState and slot props' types (#32573) @michaldudak
- &#8203;<!-- 03 -->[TabsUnstyled] Define ownerState and slot props' types (#32918) @michaldudak
- &#8203;<!-- 02 -->[TabUnstyled] Define ownerState and slot props' types (#32915) @michaldudak

### `@mui/joy@5.0.0-alpha.30`

> - &#8203;<!-- 13 -->[Joy] use `textColor` prop for Typography and Link (#32938) @siriwatknp

Reviewer 2: @siriwatknp too :D
